cli/internal/runcache/runcache.go

```diff
@@ -109,12 +109,12 @@ type TaskCache struct {
 
 // RestoreOutputs attempts to restore output for the corresponding task from the cache.
 // Returns true if successful.
```

Comment:
Update comment to account for new return value. Actually, the return documentation is already out of date and needs updating either way.